xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -111,6 +111,21 @@ void CSelectionStreams::Clear(StreamType type, StreamSource source)
   }
 }
 
+void CSelectionStreams::Clear(StreamType type, int source)
```

Comment:
As I said I want to remove **one** defective stream with the given **id**. StreamSource is an enum, with that,e.g., I can only remove all audio stream from the master file.